internal/runners/manifest/requirements.go

```diff
@@ -0,0 +1,146 @@
+package manifest
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/multilog"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/pkg/platform/api/buildplanner/model"
+	vulnModel "github.com/ActiveState/cli/pkg/platform/api/vulnerabilities/model"
+	"github.com/ActiveState/cli/pkg/platform/authentication"
+	platformModel "github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/platform/runtime/artifact"
+)
+
+type requirement struct {
+	NameOutput      string `json:"name" locale:"manifest_name,Name"`
+	VersionOutput   string `json:"version" locale:"manifest_version,Version"`
+	License         string `json:"license" locale:"manifest_license,License"`
+	Vulnerabilities string `json:"vulnerabilities" locale:"manifest_vulnerabilities,Vulnerabilities (CVEs)" opts:"omitEmpty"`
+	// Must be last of the output fields in order for our table renderer to include all the fields before it
+	NamespaceOutput string `json:"namespace" locale:"manifest_namespace,Namespace" opts:"omitEmpty,separateLine"`
+
+	// These fields are used for internal processing
+	name      string
+	namespace string
+}
```

Comment:
This is triggering my OCD a bit, the symbol naming is very inconsistent. Suggest dropping `Output` from the property suffixes, and instead adding it to the type name, to make it consistent with `requirementsOutput`.